lib/cpp/src/thrift/transport/THeaderTransport.h

```diff
@@ -39,9 +39,11 @@
 
 enum CLIENT_TYPE {
   THRIFT_HEADER_CLIENT_TYPE = 0,
-  THRIFT_FRAMED_DEPRECATED = 1,
-  THRIFT_UNFRAMED_DEPRECATED = 2,
-  THRIFT_UNKNOWN_CLIENT_TYPE = 4,
+  THRIFT_FRAMED_BINARY = 1,
+  THRIFT_UNFRAMED_BINARY = 2,
+  THRIFT_FRAMED_COMPACT = 3,
+  THRIFT_UNFRAMED_COMPACT = 4,
```

Comment:
(Moving the comment to right position as it was misleading)
@andrewcox @djwatson
Here I presume _DEPRECATED is only true for fbthrift.
Or is there any reason we should keep them and add the suffix to COMPACT ?